cmd/fluxctl/install_cmd.go

```diff
@@ -48,7 +48,7 @@ fluxctl install --git-url 'git@github.com:<your username>/flux-get-started' --gi
 		"whether to enable manifest generation")
 	cmd.Flags().StringVar(&opts.Namespace, "namespace", "",
 		"cluster namespace where to install flux")
-	cmd.Flags().BoolVar(&opts.RegistryScanning, "registry-scanning", true,
+	cmd.Flags().BoolVar(&opts.RegistryDisableScanning, "registry-scanning", true,
 		"scan container image registries to fill in the registry cache")
```

Comment:
```suggestion
		"disable scanning of container image registries")
```

Reply:
I updated it in a different way (consistent with the rest)